src/openrct2/scripting/bindings/entity/ScPeep.hpp

```diff
@@ -41,6 +41,8 @@ namespace OpenRCT2::Scripting
         { "angry", PEEP_FLAGS_ANGRY },
         { "iceCream", PEEP_FLAGS_ICE_CREAM },
         { "hereWeAre", PEEP_FLAGS_HERE_WE_ARE },
+        { "positionStatic", PEEP_FLAGS_POSITION_STATIC },
```

Comment:
I'd name it `"positionFrozen"` to keep it consistent with the other one and IMHO easier to understand what it does.